### PR TITLE
feat: create a transaction

### PR DESCRIPTION
**Pull request by thealamu**

### CODE REVIEW

File: pocketguardapp/schemas/transaction_schemas.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from pydantic import BaseModel
+
+from pocketguardapp.models.account_models import Account, AccountInfo
+from pocketguardapp.models.transaction_models import Transaction
+from .account_schemas import AccountInfoResponse
```
Comment (blacdev) [inline pocketguardapp/schemas/transaction_schemas.py:6]: Where is AccountInfoResponse?
can't seem to find it

File: pocketguardapp/services/services.py
Comment (blacdev) [inline pocketguardapp/services/services.py:1]: Kindly rename this to utility.py and let it have it own folder

File: pocketguardapp/services/transaction.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+from pocketguardapp.database.transaction import transaction_database
+from pocketguardapp.errors.error import Error
+from pocketguardapp.models.transaction_models import Transaction
+from pocketguardapp.models.account_models import AccountInfo
+from .services import parse_datetime
+
+
+TRANSACTION_KIND_INCOME = "INCOME"
+TRANSACTION_KIND_EXPENSE = "EXPENSE"
```
Comment (blacdev) [inline pocketguardapp/services/transaction.py:12]: Make these an Enum in the schema

Comment (thealamu) [reply pocketguardapp/services/transaction.py:12]: I don't see the need for any such import, these are constants that are local to the transaction service

File: pocketguardapp/services/services.py
Comment (thealamu) [reply pocketguardapp/services/services.py:1]: I generally avoid names like "util" and its variants. The tend to just collect things from everywhere overtime. It's just naming though as services.py seems to do the same thing.
The way I think of services.py is it allows the reader see exactly what context the utilities are. For me, that's a win in the long run, differentiating a general utility file from a service-only utility file

File: pocketguardapp/services/transaction.py (hunk 2 above)
Comment (thealamu) [reply pocketguardapp/services/transaction.py:12]: Unless you're referring to validation by pydantic in which case I'd say, sure let's do that but also do the check in-service. You never know who's calling your service method over what protocol

File: pocketguardapp/schemas/transaction_schemas.py (hunk 1 above)
Comment (thealamu) [reply pocketguardapp/schemas/transaction_schemas.py:6]: You can find it in the response schemas for Accounts

File: pocketguardapp/services/services.py
Comment (blacdev) [reply pocketguardapp/services/services.py:1]: Even if you are doing that, I am not comfortable with the fact that it seats in using the exact same name as the structure it is in. Kindly adjust

File: pocketguardapp/services/transaction.py (hunk 2 above)
Comment (blacdev) [reply pocketguardapp/services/transaction.py:12]: You can take the out case formatting from the service to the schema. And we can tell if we would have more types in the coming future. 

I say we move it to the schema

**Review by blacdev — APPROVED** [review]

**Pull request merged**